docs/deployment/k8s/index.mdx

```diff
@@ -8,163 +8,123 @@ wasmCloud is **compatible with, but not dependent on**, Kubernetes. We think the
+On **this** page, you can find documentation for users wishing to deploy wasmCloud with Kubernetes, including details on prerequisites such as NATS, different strategies for deploying the platform, and how to deploy the wasmCloud operator.
 
-:::info[kubectl support]
-When running wasmCloud on Kubernetes, you can use `kubectl` to deploy [wasmCloud manifests](/docs/ecosystem/wadm/model), check wasmCloud logs, and more. The example below demonstrates standard usage of `kubectl` with wasmCloud.
-:::
+## The wasmCloud platform on Kubernetes
 
-## Getting started with the wasmCloud operator
+As deployed on Kubernetes, the wasmCloud platform consists of three core parts:
 
-The wasmCloud operator can be found on GitHub at [https://github.com/wasmCloud/wasmcloud-operator](https://github.com/wasmCloud/wasmcloud-operator) and enables...
+* [wasmCloud host(s)](/docs/concepts/hosts.mdx) - Runtime environment for WebAssembly components
+* [NATS with Jetstream](/docs/ecosystem/nats/index.mdx) - Connective technology underlying the wasmCloud [lattice](/docs/concepts/lattice.mdx)
+* [wasmCloud Application Deployment Manager (`wadm`)](/docs/ecosystem/wadm/index.md) - Scheduler for application lifecycle management
 
-1. Deployment of wasmCloud Hosts onto a Kubernetes cluster via Custom Resource Definition (`WasmCloudHostConfig`)
-2. Interoperability with the [wasmCloud Application Deployment Manager (wadm)](/docs/ecosystem/wadm/) for deploying wasmcloud applications defined in the [OAM spec](https://oam.dev/) format.
-3. Automated creation of Kubernetes Services pointing to wasmCloud applications deployed with a HTTP Server capability (in the future this will become configurable&mdash;today it's automatic).
+## Deployment with Helm
 
-In order to follow the instructions on this page, you will need a Kubernetes cluster and appropriate access credentials as well as a [Helm](https://helm.sh/) installation. It will also be useful to have [wasmCloud Shell (`wash`)](/docs/installation) installed locally.
+[Helm](https://helm.sh/docs) is commonly understood as a package manager for Kubernetes, enabling platform teams to install and manage software on Kubernetes clusters. Deployments are defined in declarative [charts](https://helm.sh/docs/topics/charts/) that describe the relationship between various Kubernetes resources.
 
-You can watch a [video of the setup on YouTube](https://www.youtube.com/embed/239Q_hAKqPw?si=y0nHeFHfCPOdItNU&start=235&end=780):
+All deployments of wasmCloud on Kubernetes use Helm. We also provide an optional **wasmCloud operator** to facilitate smoother automation of the platform. Kubernetes operators automate the management of complex, stateful applications, continuously monitoring the state of **custom resources** and making real-time adjustments to align the actual state with the desired state specified by the user. 

```

Comment:
I think it's maybe more accurate to say that the wasmCloud project has chosen to embrace helm as the tool for managing wasmCloud components on Kubernetes, regardless of the deployment strategy.

Reply:
```suggestion
All deployments of wasmCloud on Kubernetes use Helm. We also provide an optional **wasmCloud Operator** to facilitate smoother automation of the platform. [Kubernetes Operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) automate the management of complex, stateful applications, continuously monitoring the state of **custom resources** and making real-time adjustments to align the actual state with the desired state specified by the user. 

```